static/app/components/modals/demoEmailModal.tsx

```diff
@@ -0,0 +1,153 @@
+import {useCallback} from 'react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import sandboxDemo from 'sentry-images/spot/sandbox.jpg';
+
+import {IconArrow} from 'sentry/icons';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import EmailForm from 'sentry/utils/demoMode/emailForm';
+import {GetUTMData, UpdateTouches} from 'sentry/utils/demoMode/utm';
+import useApi from 'sentry/utils/useApi';
+
+type Props = {
+  closeModal: () => void;
+  onAddedEmail: (email: string) => void;
+  onFailure: () => void;
+};
+
+export default function Modal({onAddedEmail, closeModal, onFailure}: Props) {
+  const api = useApi();
+
+  const handleBackClick = () => {
+    // go back to the referrer or the welcome page
+    let newUrl = 'https://sentry.io/welcome/';
+    const {referrer} = window.document;
+    // if we have a referrer and the URL is different than our current origin
+    // then we can send the back there
+    // otherwise, send them to welcome page
+    if (referrer && !referrer.startsWith(window.location.origin)) {
+      newUrl = referrer;
+    }
+
+    // track this event but if it errors out, proceed
+    try {
+      trackAnalytics('growth.email_form_pressed_back', {
+        organization: null,
+      });
+    } catch {
+      // do nothing
+    }
+
+    window.location.href = newUrl;
+  };
+
+  const handleSubmit = useCallback(
+    async (email: string) => {
+      const utmState = GetUTMData();
+
+      // always save the email before the API call
+      if (onAddedEmail) {
+        onAddedEmail(email);
+      }
+
+      try {
+        await api.requestPromise('/internal/demo/email-capture/', {
+          method: 'POST',
+          data: {
+            ...utmState.data,
+            email,
+          },
+        });
+
+        UpdateTouches(utmState);
+      } catch (error) {
+        onFailure();
+      }
+
+      closeModal();
+    },
+    [api, closeModal, onFailure, onAddedEmail]
+  );
+
+  return (
+    <div>
+      <BackButton onClick={handleBackClick}>
```

Comment:
Missing type button if we stick with the custom one.
```suggestion
      <BackButton type="button" onClick={handleBackClick}>
```